spec/index.html

```diff
@@ -625,7 +630,8 @@ <h4>The <code>&lt;<dfn id="the-map-element">map</dfn>&gt;</code> element</h4>
         <dd><code id="attr-map-lat">lat</code> — A decimal WGS84 latitude value for the map center.</dd>
         <dd><code id="attr-map-lon">lon</code> — A decimal WGS84 longitude value for the map center.</dd>
         <dd><code id="attr-map-projection">projection</code> — A case-sensitive string <a href="#tiled-coordinate-reference-systems-table">identifier</a> of the MapML coordinate reference system used by the map. Default is <a href="#tcrs-OSMTILE"><code>OSMTILE</code></a>.</dd>
-        <dd><code id="attr-map-controls">controls</code> — Show user agent map controls.</dd>
+        <dd><code id="attr-map-controls">controls</code> — Show user agent controls.</dd>
+        <dd><code>controlslist</code> — Show/hide specific user agent controls.</dd>
```

Comment:
Note that moving down the content attribute's id (and <dfn> definition) causes an inconsistency in this description list, however it seems to be the correct approach looking at how WHATWG does it. Change on other content attributes to follow. Lots of work TBD. ;)